src/Extensions/TimedScopes/ITimedScopeEventSender.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
+using System.Diagnostics;
```

Comment:
>Diagnostics [](start = 13, length = 11)

Do you really need this dependency? It's just an interface

Reply:
yes, it's required for `Activity` class